src/screens/Airdrop/RequirementsScreen.tsx

```diff
@@ -0,0 +1,147 @@
+import { StackNavigationProp } from '@react-navigation/stack';
+import React, { Component } from 'react';
+import { Image, Text, StyleSheet, View, TouchableOpacity } from 'react-native';
+import { connect } from 'react-redux';
+
+import { icons } from 'app/assets';
+import { ScreenTemplate, Button, Header } from 'app/components';
+import { Route } from 'app/consts';
+import { ApplicationState } from 'app/state';
+import { actions } from 'app/state/airdrop';
+import { typography, palette } from 'app/styles';
+
+const i18n = require('../../../loc');
+
+interface ActionProps {
+  completeThankYouFlow: Function;
```

Comment:
I think we have a warning in eslint for using a `Function`, please use full typing

Reply:
done